apps/meteor/server/startup/initialData.ts

```diff
@@ -211,7 +210,7 @@ Meteor.startup(async () => {
 	if (process.env.TEST_MODE === 'true') {
 		console.log(colors.green('Inserting admin test user:'));
 
-		const adminUser = {
+		const adminUser: Omit<IUser, 'createdAt' | 'roles' | '_updatedAt'> = {
```

Comment:
Consider using `satisfies` here. You won't have to use type casting in the follow up code.


```suggestion
		const adminUser: Omit<IUser, 'createdAt' | 'roles' | '_updatedAt'> satisfies IUser = {
```